pkg/http/handler/handler.go

```diff
@@ -18,9 +18,15 @@ type ItemsGrpcService interface {
 }
 
 type TokenService interface {
-	ParseToken(tokenPart string) error
+	ParseToken(tokenPart string) (string, error)
 }
 
+type key int
+
+const (
+	nameKey key = iota
```

Comment:
You can use key value as string to prevent using iota for values.
[Example](https://gist.github.com/cep21/a3fc8e1462d19c46422c03b0466d5869)

Reply:
Do you mean smth like this ctx := context.WithValue(req.Context(), "name", name) ? Or const ( nameKey key = string ) ?

Reply:
Any